addons/point_of_sale/views/product_combo_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_product_combo_form" model="ir.ui.view">
+        <field name="name">product.combo.form</field>
+        <field name="model">product.combo</field>
+        <field name="arch" type="xml">
+            <form string="combos">
+                <field name="name"/>
+                <field name="combo_product_ids">
+                    <tree editable="bottom">
+                        <field name="product_id"/>
+                        <field name="price"/>
+                        <field name="lst_price"/>
+                    </tree>
+                </field>
+            </form>
+        </field>
+    </record>
+
+    <record id="view_product_combo_tree" model="ir.ui.view">
+        <field name="name">product.combo.tree</field>
+        <field name="model">product.combo</field>
+        <field name="arch" type="xml">
+            <tree string="combos" create="1" delete="1">
+                <field name="name" />
+                <field name="no_of_products"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="action_product_combo" model="ir.actions.act_window">
```

Comment:
prefix the xml records with "point_of_sale". It's better for browsing in the code.